tests/PermitTests/PermitClientTests.cs

```diff
@@ -103,12 +120,8 @@ public async void TestPermitClientApi()
             Assert.True(assignedRoles.Count == (assignedRoles2.Count - 1));
 
             await permitClient.Api.DeleteUser(getUser.Key);
-            getUser = await permitClient.Api.GetUser(user.Key);
-            Assert.True(getUser == null);
-            
+
             await permitClient.Api.DeleteTenant(tenantKey);
-            getTenant = await permitClient.Api.GetTenant(getTenant.Id.ToString());
```

Comment:
why did you delete this ?

Reply:
You cannot get after delete you get 404 exception